src/app/components/login/Login.js

```diff
@@ -0,0 +1,385 @@
+import React from 'react';
```

Comment:
This file was moved but lost the diff somehow. I reconstructed the diff here: https://gist.github.com/dariusk/73b6d0546723c5f5a36d90a5dd2e026e

Looks okay to me regardless.